### ISSUE TITLE
ansible: yaml anchor and alias is not detected

### ISSUE THREAD

**Issue by markuman** [Checkmarx/kics#4630]
### Expected Behavior

`kics` must parse and respect a yaml anchor and alias.  
E.g.  `encrypted: yes` is defined in a default variable in the same file and used in every module.

### Actual Behavior

```
EBS Volume Encryption Disabled, Severity: MEDIUM, Results: 8
CIS ID: CIS Security - CIS Amazon Web Services Foundations Benchmark v1.4.0 - Rule 2.2.1
Title: Ensure EBS volume encryption is enabled
Description: Elastic Compute Cloud (EC2) supports encryption at rest when using the Elastic Block Store (EBS) service. While disabled by default, forcing encryption at EBS volume creation is supported.
Platform: Ansible
	[2]: roles/instance/tasks/detach_old_volume.yml:2
		001: - name: detache old volume
		002:   ec2_vol:
		003:     id: "{{  detach_vol_id }}"
	[3]: roles/ec2/tasks/ebs_volume.yml:49
		048:   when: param.iops is undefined and param.throughput is defined
		049:   ec2_vol:
		050:     <<: *default_parameters
```

### Steps to Reproduce the Problem

```yml
- name: >
    default volume parameters
    yaml anchors must be implemented in the same yaml document where they are used
  set_fact:
    default_parameters: &default_parameters
      instance: "{{ instance.instance_ids[0] }}"
      name: "{{ ec2_config.name }}:{{ param.device_name }}"
      kms_key_id: "{{ kms.ebs }}"
      encrypted: yes
      tags:
        Name: "{{ ec2_config.name }}:{{ param.device_name }}"
        Backup: "{{ param.backup | default('y') }}"
      modify_volume: yes
      region: "{{ region }}"

- name: ec2 volume - no iops no throughput | {{ ec2_config.name }}:{{ param.device_name }}
  when: param.iops is undefined and param.throughput is undefined
  ec2_vol:
    <<: *default_parameters
    volume_size: "{{ param.volume_size }}"
    volume_type: "{{ param.volume_type }}"
    device_name: "{{ param.device_name }}"
    delete_on_termination: "{{ param.delete_on_termination }}"
    state: "{{ param.state | default('present') }}"
  register: volstatus
  until: volstatus.failed == false
  delay: 3
  retries: 5 # maybe ec2 is slow and instance is not ready yet
  ignore_errors: yes
```

### Specifications
(N/A if not applicable)

-   Version: 1.4.9
-   Platform: docker alpine 3.15
-   Subsystem:

**Comment by rafaela-soares** [comment]
Hello, @markuman 🙂 Thank you so much for reaching us and reporting the bug!

We are happy to inform you that we are fixing it in PR #4659!
